internal/pkg/aws/ecs/ecs.go

```diff
@@ -20,6 +20,9 @@ const (
 	shortTaskIDLength      = 8
 	shortImageDigestLength = 8
 	imageDigestPrefix      = "sha256:"
+
+	// DesiredStatusStopped wraps the ecs constant DesiredStatusStopped.
```

Comment:
maybe `DesiredStatusStopped is the ECS constant DesiredStatusStopped.` since it doesn't "wrap" it.